src/databricks/labs/ucx/source_code/linters/spark_connect.py

```diff
@@ -19,18 +20,9 @@ def _cluster_type_str(self) -> str:
         return 'UC Shared Clusters' if not self.is_serverless else 'Serverless Compute'
 
     @abstractmethod
-    def lint(self, node: ast.AST) -> Iterator[Advice]:
+    def lint(self, node: NodeNG) -> Iterator[Advice]:
         pass
 
-    def lint_tree(self, tree: ast.AST) -> Iterator[Advice]:
```

Comment:
removing this is a regression. see https://github.com/databrickslabs/ucx/pull/1824

Reply:
fixed